nx.json

```diff
@@ -112,7 +112,7 @@
       "outputs": ["{projectRoot}/{options.output-dir}"],
       "options": {
         "cwd": "{projectRoot}",
-        "command": "VITE_DISABLE_ESLINT_CHECKER=true storybook build",
+        "command": "VITE_DISABLE_TYPESCRIPT_CHECKER=true VITE_DISABLE_ESLINT_CHECKER=true storybook build",
```

Comment:
What is the reason behind this change?

Reply:
a discussion about performances with @Charles. Typescript was run twice, this change make it run once